new/components/nginx/config/default.conf

```diff
@@ -1,5 +1,5 @@
 # Declares the size of a request's body. This is mainly used to allow large blueprints to be uploaded.
-client_max_body_size 50m;
+client_max_body_size 0m;
```

Comment:
A limit is a good approach. The thing is that metrics tend to be really heavy. We can change it in some future PR is needed.